### PR TITLE
Dcwc 1309 task icons and text

### PR DESCRIPTION
**Pull request by nabil386**
### Description
add icons and text to dashboard
List of proposed changes:

-
-

### CODE REVIEW

**Review by amirpourpanahi — APPROVED** [review]

**Review by amirpourpanahi — APPROVED** [review]

File: pages/dashboard.js [hunk 1]
```diff
@@ -1,134 +1,30 @@
 import BenefitCard from '../components/molecules/BenefitCard'
-
+import {
+  ACTIVE_CANADA_PENSION_PLAN,
+  SUBMITTED_CANADA_PENSION_PLAN,
+  OLD_AGE_SECURITY,
+  SUBMITTED_CPP_TASKS,
+  ACTIVE_CPP_TASKS,
```
[the remark this answers is not in the document]
Comment (nabil386) [reply pages/dashboard.js:7]: names are fixed

**Review by amirpourpanahi — APPROVED** [review]

File: contents/DashboardConstants.js [hunk 2]
```diff
@@ -0,0 +1,379 @@
+import en from '../locales/en'
+
+const t = en
+const SUBMITTED_OAS = {
```
Comment (arsanig) [inline contents/DashboardConstants.js:4]: Can we put mock data and tasks in different files in this folder?

File: tailwind.config.js [hunk 3]
```diff
@@ -54,6 +54,7 @@ module.exports = {
           light: '#F19E7A',
           solid: '#D96F41',
           dark: '#AC5028',
+          pending: '#FFA216',
```
Comment (arsanig) [inline tailwind.config.js:57]: This isn't the same yellow/orange Hexadecimal value as in the Figma, can you please mate it to the Figma?

File: components/molecules/BenefitTasks.test.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
+import { render, screen } from '@testing-library/react'
```
[the remark this answers is not in the document]
Comment (nabil386) [reply components/molecules/BenefitTasks.test.js:1]: this test should be in this PR

File: components/atoms/VerticalStepper.test.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
+import { render, screen } from '@testing-library/react'
```
[the remark this answers is not in the document]
Comment (nabil386) [reply components/atoms/VerticalStepper.test.js:1]: this must have came from dashboard branch merge

File: tailwind.config.js (hunk 3 above)
Comment (nabil386) [reply tailwind.config.js:57]: this is not part of this PR is must have came from dashboard branch merge

**Pull request merged**